sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/OracleDatabaseIT.java

```diff
@@ -33,12 +33,10 @@ protected RestService getApp() {
         return app;
     }
 
-    // TODO: Remove after https://github.com/quarkus-qe/quarkus-test-framework/issues/503 is resolved
```

Comment:
Can we use `OracleService` instead of `ProvisionalOracleService` now that issue is fixed?

Reply:
Yes, it has the same implementation now. I will rewrite this